resources/views/contacts.blade.php

```diff
@@ -0,0 +1,5 @@
+@extends('layouts.app')
+
+@section('content')
+<h1>Contacts page</h1>
```

Comment:
Can we change the h1 content to just Contact us?

Reply:
ok working on it!